pkg/snapshot/snapshot.go

```diff
@@ -0,0 +1,57 @@
+package snapshot
+
+import (
+	"context"
+
+	"github.com/lima-vm/lima/pkg/driver"
+	"github.com/lima-vm/lima/pkg/driverutil"
+	"github.com/lima-vm/lima/pkg/store"
+)
+
+func Del(ctx context.Context, inst *store.Instance, tag string) error {
+	y, err := inst.LoadYAML()
+	if err != nil {
+		return err
+	}
+	limaDriver := driverutil.CreateTargetDriverInstance(&driver.BaseDriver{
+		Instance: inst,
+		Yaml:     y,
+	})
+	return limaDriver.DeleteSnapshot(ctx, inst.Status == store.StatusRunning, tag)
+}
+
+func Save(ctx context.Context, inst *store.Instance, tag string) error {
+	y, err := inst.LoadYAML()
+	if err != nil {
+		return err
+	}
+	limaDriver := driverutil.CreateTargetDriverInstance(&driver.BaseDriver{
+		Instance: inst,
+		Yaml:     y,
+	})
+	return limaDriver.CreateSnapshot(ctx, inst.Status == store.StatusRunning, tag)
```

Comment:
Can we do this inst.status check in the qemu driver itself (we do have access to instance in driver)??

This way we don't need to expose a bool arg in driver interface

Reply:
Sure, I will remove it from the "API"